tests/core/pm-module/test_ens_integration.py

```diff
@@ -141,3 +141,13 @@ def test_web3_ens(ens):
     assert pkg_name == 'owned'
     assert version == '1.0.0'
     assert manifest_uri == 'ipfs://QmcxvhkJJVpbxEAa6cgW3B6XwPJb79w9GpNUv2P2THUzZR'
+
+
+def test_ens_inherits_web3_middlewares(w3):
```

Comment:
nit: this ethpm directory doesn't feel like the best home for this test. i don't see another natural place for ENS-specific integration tests. choose your own adventure.